packages/component-utilities/src/isEmptyDataset.js

```diff
@@ -0,0 +1,7 @@
+export default function isEmptyDataset(data) {
+	if (data === undefined || data[0] === undefined || data.length === 0) {
```

Comment:
We may want to have to guard against nulls or falsey values
```suggestion
	if (!data || !data[0] || !data?.length) {
```